elastica/boundary_conditions.py

```diff
@@ -5,6 +5,7 @@
     "FreeRod",  # Deprecated: remove v0.3.0
     "OneEndFixedBC",
     "OneEndFixedRod",  # Deprecated: remove v0.3.0
+    "ConfigurableFixedConstraint",
```

Comment:
Since we allow some dofs to be free, fixed keyword doesn't gel well, maybe 
```suggestion
    "ConfigurableConstraint",
```

Reply:
Changed in https://github.com/GazzolaLab/PyElastica/pull/143/commits/4a9c71e4faa98ca2e35e461a9b3f79c1edcfa2d7